tests/unit/multipleBackend/BackendInfo.js

```diff
@@ -7,26 +7,32 @@ const log = new DummyRequestLogger();
 const dummyBackendInfo = new BackendInfo('mem', 'file', '127.0.0.1');
 
 describe('BackendInfo class', () => {
-    describe('isValidControllingBackendParam', () => {
+    describe('controllingBackendParam', () => {
         it('should return false if objectLocationConstraint is invalid', () => {
```

Comment:
I would change to say, "should return object with applicable error if objectLocationConstraint is invalid"